src/java-postgres/README.md

```diff
@@ -7,7 +7,7 @@ Develop applications with Java and PostgreSQL. Includes a Java application conta
 
 | Options Id | Description | Type | Default Value |
 |-----|-----|-----|-----|
-| imageVariant | Java version (use -bookworm, or -bullseye variants on local arm64/Apple Silicon): | string | 21-bullseye |
```

Comment:
Latest Java LTS is 25. I suggest documenting with the latest GA and LTS version.